src/util.rs

```diff
@@ -130,6 +130,11 @@ pub fn serialize_signature(signature: &secp256k1::recovery::RecoverableSignature
     signature_bytes
 }
 
+pub fn blake160(message: &[u8]) -> bytes::Bytes {
+    let r = ckb_hash::blake2b_256(message);
+    bytes::Bytes::copy_from_slice(&r[..20])
+}
```

Comment:
Return `H160` here. Since `H160` have more rich information (the data is 20 bytes)